examples/federated-learning/main.py

```diff
@@ -52,6 +52,9 @@
                       "simulate the training by splitting the data and "
                       "distributing it amongst the data owners. "
                       "Only applicable for local computations."))
+flags.DEFINE_string("save_dir", None,
+                    ("Where the model (h5 format) should be saved. If None the "
```

Comment:
suggest passing the format as an additional parameter that defaults to `h5` -- can change the name of this one to `save_path` to account for the difference between a saved model dir & h5 file in the different cases